src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-cluster-form/nfs-cluster-form.component.html

```diff
@@ -0,0 +1,40 @@
+
+  <div cdsCol
+       [columnNumbers]="{md: 4}">
+  <ng-container *cdFormLoading="loading">
+  <form name="nfsForm"
+        [formGroup]="nfsForm"
+        #formDir="ngForm"
+        novalidate>
+  <div i18n="form title"
+       class="form-header">
+  <span *ngIf="isEdit">
+          Set Cluster QOS(Quality of Service)
```

Comment:
let's just call it `Quality of Service` instead.
```suggestion
          Set Cluster Quality of Service
```